skytemple_files/_resources/skytemple_pmdsky_debug_symbols.yml

```diff
@@ -0,0 +1,93 @@
+# Symbols that are too specific to SkyTemple or related mods to be included in pmdsky-debug.
+# Or they are values in the middle of functions.
+
+arm9:
+  functions:
+    - name: DebugPrint2
+      address:
+        NA: 0x200C30C
+        EU: 0x200C284  # TODO: Is this really the same as DebugPrint0 for EU???
+      description: Would log a printf format string in the debug binary. A no-op in the final binary.
+
+  data:
+    - name: JUICE_BAR_NECTAR_IQ_GAIN
+      address:
+        NA: 0x2011810
+        EU: 0x20118B8
+      length:
+        NA: 0x1
+        EU: 0x1
+      description: "IQ gain when ingesting nectar at the Juice Bar. Note: This is a value in the middle of a function."
+
+    - name: COMPRESSED_IQ_GROUP_SKILLS
+      address:
+        NA: 0x20A1D90
+        EU: 0x20A2314
+      length:
+        NA: 0x90
+        EU: 0x90
+      description: Replaces IQ_GROUPS_SKILLS when the patch "CompressIQData" is applied.
+
+    - name: GUEST_MONSTER_DATA2
+      address:
+        NA: 0x204F148
+        EU: 0x204EE10
+      length:
+        NA: 0x1D0
+        EU: 0x1D0
+      description: Requires EditExtraPokemon patch.
+
+    - name: EXTRA_DUNGEON_DATA
+      address:
+        NA: 0x204EFE0
+        EU: 0x204ECA8
+      length:
+        NA: 0x168
+        EU: 0x168
+      description: Requires EditExtraPokemon patch.
+
+overlay0:
+  data:
+    - name: MAIN_MENU_SONG_ID
+      address:
+        NA: 0x22BE1A0
+        EU: 0x22BE9B4
+      length:
+        NA: 0x1
+        EU: 0x1
+      description: "Song playing in the main menu. Note: Probably in the middle of a function."
+
+overlay9:
+  data:
+    - name: MAIN_MENU_RETURN_SONG_ID
```

Comment:
This one's already in `literals.yml` as `TOP_MENU_RETURN_MUSIC_ID`. Let's add the description upstream.